docs/zed/api.md

```diff
@@ -44,13 +40,55 @@ POST /pool
 | Name | Type | In | Description |
 | ---- | ---- | -- | ----------- |
 | name | string | body | **Required.** Name of the pool. Must be unique to lake. |
-| layout.order | string | body | Order of value storage in pool. Possible values: desc, asc. Default: asc. |
-| layout.keys | [string] | body | Primary key(s) of pool. Default: ts. |
+| layout.order | string | body | Order of storage by primary key(s) in pool. Possible values: desc, asc. Default: asc. |
+| layout.keys | [[string]] | body | Primary key(s) of pool. The element of each inner string array should reflect the hierarchical ordering of named fields within indexed records. Default: [[ts]]. |
```

Comment:
As pointed out by @jameskerr when I asked about this on Slack, I suppose this could be simplified to accept array elements that are mere strings if the user is seeking to treat a top-level field as a key.